### ISSUE TITLE
0.12.2, com.kazurayam.materialstore.base.report.MaterialProductGroupReporterTest produced an invalid report

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#352]
it produced an empty report
<img width="1552" alt="スクリーンショット 2022-11-13 18 11 43" src="https://user-images.githubusercontent.com/4467417/201514407-e53852dd-2384-4bb4-a097-60595f61a12a.png">

**Comment by kazurayam** [comment]
This test uses a fixture named "issue#80#.

```
private static final Path fixtureDir =
            Paths.get(".").resolve("src/test/fixtures/issue#80");

```

The 2 test classes reported by #350 also uses the same fixture.

So, the fixute "issue#80" could be the cause of problems.

But how?

**Comment by kazurayam** [comment]
fixed at 0.12.5
